src/libraries/System.Private.Xml/src/System/Xml/XmlConvert.cs

```diff
@@ -1645,5 +1645,174 @@ internal static Exception CreateInvalidNameCharException(string name, int index,
         {
             return CreateException(index == 0 ? SR.Xml_BadStartNameChar : SR.Xml_BadNameChar, XmlException.BuildCharExceptionArgs(name, index), exceptionType, 0, index + 1);
         }
+
+        internal static bool TryFormat(bool value, Span<char> destination, out int charsWritten)
+        {
+            ReadOnlySpan<char> valueSpan = value ? "true" : "false";
+
```

Comment:
If the goal is to be as fast as possible, take a look at [Boolean.TryFormat](https://github.com/dotnet/runtime/blob/69045adc0934c9f0ac2d04c57da8ce1acab0a0ad/src/libraries/System.Private.CoreLib/src/System/Boolean.cs#L97-L123) and adjust for the lower case T and F. If you don't want to go that far, don't cast to ReadOnlySpan as String has a TryCopyTo method that you can use and avoid executing the code in the implicit case.

Reply:
@mconnew  I receive feedback from owners, that they prefer in this method readability over performance.
Thank you for suggestion of using string.TryCopyTo - I used it.